code/game/objects/items/holy_weapons.dm

```diff
@@ -462,8 +414,7 @@
 	hitsound = 'sound/weapons/chainsawhit.ogg'
 	tool_behaviour = TOOL_SAW
 	toolspeed = 0.5 //faster than normal saw
-	chaplain_spawnable = FALSE
```

Comment:
I think you should keep the chaplain_spawnable variable. It's unclear that you have to set the menu_description to null to tell if the nullrod type is spawnable, even if it is in the documentation.

Reply:
people should be reading the documentation though, if someone is wondering how weapons are made spawnable or not they'd simply read what i've docced, no?

Reply:
Variable names should usually be just as clear as the documentation.

Reply:
okay, i'll change it 👍